webpack.config.babel.js

```diff
@@ -0,0 +1,76 @@
+import HtmlWebPackPlugin from 'html-webpack-plugin'
+import webpack from 'webpack'
+import DashboardPlugin from 'webpack-dashboard/plugin'
+import InlineChunkHtmlPlugin from 'react-dev-utils/InlineChunkHtmlPlugin'
+import ForkTsCheckerWebpackPlugin from 'fork-ts-checker-webpack-plugin'
+
+const isProduction = process.env.NODE_ENV == 'production'
+const baseUrl = isProduction && process.env.CI ? '/when-hosting-not-at-root/' : '/'
```

Comment:
What is `/when-hosting-not-at-root`?

Reply:
it's a baseurl for webpack to modify filepaths as appropriate
for now it's a placeholder